### DISCUSSION TITLE
In Userprofile Textarea is not working still shows a textbox

### DISCUSSION THREAD

**Discussion by parthbulusu** [keycloak/keycloak#33595] [Archive]
RealmSettings>UserProfile>CreateAttribute
Select Annotation 'InputType: TextArea' Also addeds cols and rows
The attribute under user is expected to show a textarea, still showing text box

**Comment by ssilvert** [comment]
I think there is already an issue for this bug.  I suggest you search the issues and see if it has been fixed yet.  If not, create a new issue.

https://github.com/keycloak/keycloak/issues
